styles/views/control-window.css

```diff
@@ -0,0 +1,60 @@
+.controlWindow.square{
+  border-radius: 30px;
+}
+
+.controlWindow.circle{
+  border-radius: 100%;
+}
+
+.controlWindow.square.back{
+  background-color: rgba(166,166,166,1);
+  width: 320px;
+  height: 320px;
+  position: absolute;
+}
+
+.controlWindow.circle.front{
```

Comment:
naming is confusing again. is this the css for the view or the window? In addition, making the class specifiers siblings make the html messy. ie. Convert the `.controlWindow.circle.front` to `.controlWindow .circle .front`, and nest the html appropriately, as right now the html for the view looks messy since they all contain the same class